src/main/java/com/fundation/logic/model/search/ImageSearch.java

```diff
@@ -79,19 +80,24 @@ public List searchInPath(String path) {
                     if (criteriaHeight.length() != 1) {
                         height = MetadataImageExtractor.getHeight();
                     }
+                    String colorSpace = "All";
+                    if (criteriaColorSpace != "All"){
+                        colorSpace = MetadataImageExtractor.getSearchColorSpace();
+                    }
                     Date creationDate = FileInfo.getFileDate(fileExtractor, "creation");
                     Date accessDate = FileInfo.getFileDate(fileExtractor, "access");
                     Date modificationDate = FileInfo.getFileDate(fileExtractor, "modification");
                     Float fileSize = FileInfo.getFileSize(fileExtractor);
                     if (evaluateString(fileName, criteriaFileName)
                             && evaluateString(fileExtension, criteriaExtension)
                             && evaluateString(width, criteriaWidth)
-                            && evaluateString(height, criteriaHeight)) {
-                        List<String> metadata = MetadataCommonExtractor.getSearchListMetadata();
+                            && evaluateString(height, criteriaHeight)
+                            &&evaluateString(colorSpace,criteriaColorSpace)) {
+                        List<String> metadata = MetadataImageExtractor.getSearchListMetadata();
                         CustomizedFile matchingFile = new CustomizedFile(fileExtractor.getAbsolutePath(),
                                 fileName, fileExtension, false, false,
                                 fileSize, creationDate, accessDate,
-                                modificationDate, "MimeType", "video", metadata);
+                                modificationDate, owner, "mime type", metadata);
```

Comment:
The String "mime type" should be a variable

Reply:
done